tools/js-sdk-release-tools/src/test/codeUpdate/testCases/tsp-location.yaml

```diff
@@ -0,0 +1 @@
+repo: Azure/azure-rest-api-specs
```

Comment:
@weidongxu-microsoft just to clarify I notice for Java in tsp-location.yaml the repo value is starting with `Azure/azure...` not relevant path `../azure-rest-api-spec`, do we manually update the value after tsp cli?